index.css

```diff
@@ -365,100 +379,88 @@ main {
 
 .aboutme__background-image {
   display: flex;
-  width: 375px;
-  height: 250px;
-  background-image: url('./images/aboutme.png');
   background-repeat: no-repeat;
   background-position: right;
 }
 
-.aboutme__separator {
-  margin: 50px 0 0 0;
-  width: 375px;
-  justify-content: center;
-  align-items: center;
-}
-
-.aboutme__separator-line {
-  height: 2px;
-  background-image: url('./images/aboutme_separator.png');
-}
-
 .aboutme__details {
-  padding-top: 70px;
-  width: 375;
   background: #1c1a19;
   color: #fff;
   display: flex;
   flex-direction: column;
-  align-items: center;
+  padding-bottom: 10%;
 }
 
-.grid-container {
-  display: grid;
-  grid-template-columns: 30px 200px;
-  grid-gap: 10px;
-  padding: 10px;
-  text-align: center;
-  font-family: 'InaiMathi', sans-serif;
+.aboutme__sub-container {
```

Comment:
- [ ] Kindly check your skills section, there's a problem with the width on your containers since here the width is bigger than the screen and this result in a scroll horizon possible. This brings a bad UX.
- [ ] Kindly check your skills sections, all have the same title, change them to match with Figma design and also position them in the center


![image](https://user-images.githubusercontent.com/93630700/213793751-6f5055c1-3eaa-4b6c-8f78-21f120d9afce.png)